build-docker-image/action.yaml

```
@@ -146,7 +146,7 @@ runs:
           type=schedule,pattern=nightly
           type=edge,branch=develop
           type=ref,event=pr
-          type=sha,prefix={{branch}}-
+          type=ref,prefix=,suffix=-{{sha}},event=branch
```

We should remove the prefix here?
```suggestion
          type=ref,suffix=-{{sha}},event=branch
```